scenes/enemies/Sheep/Sheep.gd

```diff
@@ -0,0 +1,58 @@
+extends JEnemyBody2D
+
+@onready var sprite := $Sprite2D
+@onready var anim_player := $AnimationPlayer
+@onready var floating_text_scene = preload("res://scenes/templates/JFloatingText/JFloatingText.tscn")
+@onready var avoidance_rays := $AvoidanceRays
+@onready var beehave_tree := $BeehaveTree
+var behavior: Node2D
+
+var is_dead := false
+
+func _ready():
+	super()
+	if J.is_server():
+		beehave_tree.enabled = true
+	enemy_class = "Sheep"
+	stats.movement_speed = 50
+	stats.max_hp = 100
```

Comment:
I can definitely do that. I _started_ to do it, but then I wondered if there would be scenarios such that the max_hp needs to change without updating the current health. I'll change it for now though as I think this is the most common case.

Reply:
I was thinking when you change equipment that has effect on your max hp, you might not want to set your hp to max hp as it can be abused as insta heal.